src/controllers/expenses.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+const expenseService = require('./../services/expenses');
+const userService = require('./../services/users');
+
+const getAll = (req, res) => {
+  const filters = req.query;
+  const filteredExpenses = expenseService.getAll(filters);
+
+  res.send(filteredExpenses);
+};
+
+const getOne = (req, res) => {
+  const { expenseId } = req.params;
+  const foundExpense = expenseService.getOne(expenseId);
+
+  if (!foundExpense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(foundExpense);
+};
+
+const create = (req, res) => {
+  const data = req.body;
+  const { name, userId } = req.body;
+
+  if (name === 'undefined') {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const foundUser = userService.getOne(userId);
+
+  if (!foundUser) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newExpense = expenseService.create(data);
```

Comment:
what if in the data we get not enough parameters for a new expense?

Reply:
done